static/js/common.js

```diff
@@ -1,10 +1,5 @@
 import $ from "jquery";
 
-// This reloads the module in development rather than refreshing the page
-if (module.hot) {
-    module.hot.accept();
```

Comment:
(This enabled hot reloading for just this one file and no other files, which doesn’t seem worthwhile. A developer can always temporarily add `module.hot.accept()` to a particular file they want to hot-reload while they’re actively editing it.)